client/pages/components/PassTextBox.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import store from "../../store"
+import Tippy from '@tippy.js/react';
+import 'tippy.js/dist/tippy.css';
+
+export default function PassTextBox({ label}) {   
```

Comment:
PassTextBox (I assume) is only one component down from the Login component?  
```
Login
    PassTextBox
    Etc
```

Where Login sends the POST request to the server?

If this is the case you could manage the state in the Login component, and pass down the current state, and the function to change the current state.